src/sentry/issues/services/issue/impl.py

```diff
@@ -45,3 +46,25 @@ def upsert_issue_email_reply(
         # Call the task synchronously so that the outbox retry works
         # correctly should this fail.
         process_inbound_email(from_email, group_id, text)
+
+    def get_linked_issues(
```

Comment:
could be better to specify this naming/function is specifically for integrations (?)

Reply:
Yeah let me update that. Originally it didn't have integrations in the query, but turns out it was needed because external issue keys aren't unique.